index.html

```diff
@@ -45,77 +44,148 @@ <h1 class="heading">TheYalow</h1>
+                transform="translate(-8 -319)" />
+            </svg></a></li>
         <li><a href="#"><svg xmlns="http://www.w3.org/2000/svg" width="13" height="19" viewBox="0 0 13 19"
               class="slider-arrow">
-            <path id="Shape_1_copy" data-name="Shape 1 copy" class="cls-1"
-              d="M1291.99,327.2l-2.96,2.985-0.01,0-6.29,6.333a2.1,2.1,0,0,1-2.81-.129,2.119,2.119,0,0,1-.28-2.734l6.28-6.314-6.6-6.039s-0.79-1.762.19-2.764,2.77-.22,2.77-0.22l6.6,6.042,0.02-.02Z"
-              transform="translate(-1279 -318)" />
-          </svg></a></li>
+              <path id="Shape_1_copy" data-name="Shape 1 copy" class="cls-1"
+                d="M1291.99,327.2l-2.96,2.985-0.01,0-6.29,6.333a2.1,2.1,0,0,1-2.81-.129,2.119,2.119,0,0,1-.28-2.734l6.28-6.314-6.6-6.039s-0.79-1.762.19-2.764,2.77-.22,2.77-0.22l6.6,6.042,0.02-.02Z"
+                transform="translate(-1279 -318)" />
+            </svg></a></li>
       </ul>
+      <div class="shadow"></div>
     </section>
   </div>
 
   <section class="features">
-  <div class="features-wrapper">
-    <div class="features-item"><img src="assets/Shape 2.png" alt="">
-      <p>better design</p>
-    </div>
-    <div class="features-item"><img src="assets/Shape 3.png" alt="">
-      <p>customise</p>
-    </div>
-    <div class="features-item"><img src="assets/Shape 4.png" alt="">
-      <p>Its Free</p>
+    <div class="features-wrapper">
+      <div class="features-item"><img src="assets/shape2.png" alt="" class="shape2">
+        <p>better design</p>
+      </div>
+      <div class="features-item"><img src="assets/shape3.png" alt="" class="shape3">
+        <p>customise</p>
+      </div>
+      <div class="features-item"><img src="assets/shape4.png" alt="" class="shape4">
+        <p>Its Free</p>
+      </div>
     </div>
-  </div>
   </section>
 
-  <div class="news_container">
-    <h2>Lorem ipsum dolor sit amet</h2>
-    <div class="news">
-      <div class="news_content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed d</div>
-      <div class="media_news"><video src="https://www.youtube.com/watch?v=RaQp6oM-8_8" poster="assets/poster2.jpg"
-          controls></video></div>
-    </div>
-    <div class="news">
-      <div class="news_content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed d</div>
-      <div class="news_video"></div>
-    </div>
-    <div class="date">
-      <p class="date_day">25</p>
-      <p class="date_month">april</p>
-    </div>
+  <div class="news-container">
+    <section class="news">
+      <h2 class="heading-2">Lorem ipsum dolor sit amet</h2>
+      <div class="news-wrapper">
+        <div class="news-item">
+          <div class="news-content">
+              <p class="news-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
+                inci-<br>didunt ut</p>
+                <p class="news-text-640">Lorem ipsum dolor sit amet, consecte-<br>tur adipisicing elit, sed do eiusmod tempor
```

Comment:
Often string is inserted to the html template via some API, in this case if you want new line you won't know where to split it using `br`, so prefer other ways to create new line